src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -73,28 +90,45 @@ const InstanceSnapshots: FC<Props> = ({ controlTarget, instance }) => {
 
   return (
     <>
-      {controlTarget &&
-        createPortal(
-          <>
-            <CreateSnapshotBtn openSnapshotForm={() => setModalOpen(true)} />
-          </>,
-          controlTarget
-        )}
       {isModalOpen && (
         <CreateSnapshotForm
           instance={instance}
           close={() => setModalOpen(false)}
         />
       )}
       <Row>
-        {instance.snapshots && instance.snapshots.length > 0 ? (
+        <Col size={6}>
+          {hasSnapshots && (
+            <SearchBox
+              className="search-box margin-right"
+              type="text"
+              onChange={(value) => {
+                setQuery(value);
+              }}
+              placeholder="Search for snapshots"
+              value={query}
+              aria-label="Search for snapshots"
+            />
```

Comment:
I think the input is a little too wide on wide screens, also the responsive behaviour seems a bit off.

Reply:
@al-bakalova what is the expected responsive behaviour here?

Reply:
Please make sure to always fit the search and the Create new button on the same line.
As discussed earlier, please reuse the max width from the Instance header.